packages/wallets/src/evm/connectors/smart-wallet/index.ts

```diff
@@ -69,6 +70,7 @@ export class SmartWalletConnector extends Connector<SmartWalletConnectionArgs> {
             this.config.secretKey,
           ),
       gasless: config.gasless,
+      doNotDeployOnSignMessage: doNotDeployOnSignMessage,
```

Comment:
can we do a positive instead of negative here?

`deployOnSign` and defaults to true if not set or explicitely set to false

Reply:
kinda implicitly saying not recommended/you're actively taking the decision not to deploy and how I named it in Unity lol

Reply:
renamed in https://github.com/thirdweb-dev/js/pull/2222/commits/6226a1973a492ee2a2369333f8c8d13bf8414812